packages/library/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = (neutrino, opts = {}) => {
       presets: [
         [require.resolve('@babel/preset-env'), {
           debug: neutrino.options.debug,
+          forceAllTransforms: true,
```

Comment:
I don't think we want `forceAllTransforms` for the same reasons as mentioned above

Reply:
so what are those targets then?

Reply:
CRA uses `ie9` to compile for ES5.

Reply:
do there need to be targets for other browsers? i thought you needed to specify them for each type

Reply:
For the browser targets, Babel essentially reduces support down to the least common denominator. With `ie9` being pretty low, it outputs code to the least common denominator of ES5.

Reply:
All that to say, by specifying `ie9`, all the output code is ES5 which all of the other browsers will happily use.